### PR TITLE
doc: document the relevance of mds_namespace mount option

### PR DESCRIPTION
**Pull request by joscollin**
Fixes: https://tracker.ceph.com/issues/57673

### CODE REVIEW

File: doc/cephfs/mount-using-kernel-driver.rst [hunk 1]
```diff
@@ -103,6 +103,26 @@ To mount a subtree of the CephFS root, append the path to the device string::
...
+The key-value argument right after option ``-o`` is CephX credential;
+``name`` is the username of the CephX user we are using to mount CephFS.
+
+To mount a non-default FS ``cephfs2``, in case the cluster has multiple FSs::
+
+    mount -t ceph :/ /mnt/mycephfs -o name=admin,fs=cephfs2
+
+    or
+
+    mount -t ceph :/ /mnt/mycephfs -o name=admin,mds_namespace=cephfs2
```
Comment (lxbsz) [inline doc/cephfs/mount-using-kernel-driver.rst:124]: Hi Jos,

The `mds_namespace=` has been deprecated. Please add one comment to tell users use `fs=` instead.

**Comment by lxbsz** [comment]
Jos, IMO we should add the doc for `man/8/mount.ceph.rst` ?

**Comment by joscollin** [comment]
> Jos, IMO we should add the doc for `man/8/mount.ceph.rst` ?

I think we should keep the man pages as is, with the new syntax. 
The other comment is addressed.

File: doc/cephfs/mount-using-kernel-driver.rst [hunk 2]
```diff
@@ -103,6 +103,28 @@ To mount a subtree of the CephFS root, append the path to the device string::
...
+
+The option ``mds_namespace`` is deprecated. Use ``fs=`` instead when using the old syntax for mounting.
```
Comment (dparmar18) [inline doc/cephfs/mount-using-kernel-driver.rst:126]: this could be wrapped inside a `.. note::`?

**Comment by vshankar** [comment]
> > Jos, IMO we should add the doc for `man/8/mount.ceph.rst` ?
> 
> I think we should keep the man pages as is, with the new syntax. The other comment is addressed.

Its makes sense to make a note in the man page too.

**Comment by vshankar** [comment]
LGTM (after addressing comments)

**Comment by joscollin** [comment]
Comments addressed. @vshankar Please check.

**Review by dparmar18 — APPROVED** [review]
LGTM

**Review by vshankar — APPROVED** [review]

**Pull request merged**